framework/include/problems/FEProblemBase.h

```diff
@@ -1724,8 +1724,13 @@ class FEProblemBase : public SubProblem, public Restartable
 
   /**
    * Update data after a mesh change.
+   * Iff intermediate_change is true, only perform updates as
+   * necessary to prepare for another mesh change
+   * immediately-subsequent. If \p changed_throug_amr then we will
```

Comment:
```suggestion
   * immediately-subsequent. 
```
It's not a parameter. Best describe the parameters. Then give advice on how to use it when in the context of AMR